src/atom_finder/classifier/implicit-predicate.clj

```diff
@@ -0,0 +1,38 @@
+(in-ns 'atom-finder.classifier)
+(import '(org.eclipse.cdt.core.dom.ast  IASTIfStatement IASTForStatement IASTWhileStatement IASTDoStatement IASTBinaryExpression))
+
+(def logical-operators #{IASTBinaryExpression/op_equals

```

Comment:
This list looks like it might be useful in other contexts as well. Could you put this in classifier_util.clj??